src/code42cli/cmds/users.py

```diff
@@ -401,12 +402,16 @@ def _get_role_id(sdk, role_name):
 
 
 def _get_users_dataframe(sdk, columns, org_uid, role_id, active):
-    users_generator = sdk.users.get_all(active=active, org_uid=org_uid, role_id=role_id)
-    users_list = []
-    for page in users_generator:
-        users_list.extend(page["users"])
-
-    return DataFrame.from_records(users_list, columns=columns)
+    try:
+        users_generator = sdk.users.get_all(
+            active=active, org_uid=org_uid, role_id=role_id
+        )
+        users_list = []
+        for page in users_generator:
+            users_list.extend(page["users"])
+        return DataFrame.from_records(users_list, columns=columns)
+    except Py42OrgNotFoundError:
```

Comment:
We can just add `Py42OrgNotFoundError` to the list of exceptions we handle in `code42cli.click_ext.groups.ExceptionHandlingGroup` then we don't need to handle it specifically in each command.

Reply:
Then we can update py42 to correctly raise a `Py42OrgNotFoundError` from `sdk.devices.get_page()` (and maybe a few other custom ones where a bad value could be passed). Then just maybe wait to release the CLI until the py42 issue has been fixed.

Reply:
Yep, that sounds like the right move. I'll remove this handling from the device list fetch and open a PR against py42 to properly raise the exception there.